flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/table/validation/CalcValidationTest.scala

```diff
@@ -108,7 +108,7 @@ class CalcValidationTest extends TableTestBase {
   def testDropColumnsWithValueLiteral(): Unit = {
     val util = streamTestUtil()
     val tab = util.addTableSource[(Int, Long, String)]("Table3",'a, 'b, 'c)
-    tab.dropColumns("'a'")
+    tab.dropColumns("a")
```

Comment:
why is this still a string and not a ref?

Reply:
This is actually correct, as it's testing that the string literal doesn't work